__test__/cypress/plugins/utils.js

```diff
@@ -8,15 +8,14 @@ export async function getOrCreateTestOrganization() {
     .knex("organization")
     .where("name", DEFAULT_ORGANIZATION_NAME)
     .first();
-  if (!org) {
-    org = await r
-      .knex("organization")
-      .insert({
-        name: DEFAULT_ORGANIZATION_NAME,
-        uuid: uuid.v4(),
-        features: JSON.stringify({ EXPERIMENTAL_PHONE_INVENTORY: true })
-      })
-      .returning("*");
```

Comment:
This code path was failing with an error every time in CI. `returning(*)` doesn't return an object. It was never being hit in development because these tests were never run against an empty database.